steps/python/python.build.yml

```diff
@@ -2,46 +2,46 @@
 #
 # Build, lint and test Python modules based on setuptools (setup.py)
 #
-
+---
```

Comment:
Sometimes we have a newline after that, sometimes not. The formatter is not checking that?

Reply:
No, unfortunately it is not. I will try to make them consistent manually.

Reply:
Fixed. We might look into whether we can add a rule for this in the future.